README.md

```diff
@@ -31,13 +31,19 @@ on:
     branches: [ main ]
 jobs:
   Pull-request-files-check:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest]
```

Comment:
This is nice to have on the action's test, so that we know the checks run correctly on all platforms.
I would suggest changing the documentation to tell users to use it on one platform only though :)
(ubuntu-latest being the best choice as it costs less to run things on linux in github actions)

Reply:
OK